app/controllers/v1/sessions_controller.rb

```diff
@@ -405,7 +406,16 @@ def conditional_log_message_to_sentry(message, level, context)
       if invalid_message_timestamp_error?(message)
         Rails.logger.warn("SessionsController version:v1 context:#{context} message:#{message}")
       else
-        log_message_to_sentry(message, level, extra_context: context)
+        full_message = "#{LOG_PREFIX} #{message}"
```

Comment:
I think we can get rid of this entire `conditional_log_message_to_sentry` method. Based on the comment it was to filter out errors they didn't want sent to sentry. 

We can just replace the `conditional_log_message_to_sentry` method call above with this log
```ruby
Rails.logger.warn("[SessionsController] version:v1 message:#{message}", context:)
```